.github/workflows/ubuntu.yml

```diff
@@ -89,8 +83,9 @@ jobs:
       - name: Run configure
         env:
           arch: ${{matrix.arch}}
+          configure: ${{matrix.configure}}
         run: >-
-          $SETARCH ../src/configure -C --disable-install-doc ${{ matrix.configure }}
+          $SETARCH ../src/configure -C --disable-install-doc ${configure:cppflags=-DRUBY_DEBUG}
```

Comment:
Maybe?
```suggestion
          $SETARCH ../src/configure -C --disable-install-doc ${configure:-cppflags=-DRUBY_DEBUG}
```

Reply:
Ah yes, that's what I intended but somehow I made a typo. Thanks!